taxjar/client.py

```diff
@@ -164,9 +168,19 @@ def _request(self, method, endpoint, data=None):
     def _uri(api_url, endpoint):
         return api_url + endpoint
 
+    @staticmethod
+    def _get_user_agent():
+        platform = ' '.join(os.uname())
+        python_version = '.'.join((str(i) for i in sys.version_info[0:3]))
+        try:
```

Comment:
I would add a new line before the `try:` and before the `return`.

Reply:
I did consider it, but we're not following that pattern anywhere else in the repo.